src/LoginServer/Network/LoginPacketHandler.cs

```diff
@@ -185,6 +186,8 @@ public void CB_BARRACKNAME_CHANGE(LoginConnection conn, Packet packet)
 			LoginServer.Instance.Database.UpdateTeamName(conn.Account.Id, name);
 
 			Send.BC_BARRACKNAME_CHANGE(conn, TeamNameChangeResult.Okay);
+			Send.BC_ACCOUNT_PROP(conn, conn.Account);
+			Send.BC_NORMAL_Run(conn, "THEMA_BUY_SUCCESS");
```

Comment:
I considered moving strings like "THEMA_BUY_SUCCESS" into a separate location. I think it would be better to statically define them somewhere so that they can be updated only once instead of multiple places.

Reply:
Yes, definitely. Though the main advantage I see is auto-completion and protection from typos.